src/test/kotlin/no/nav/faktureringskomponenten/service/beregning/BeløpBeregnerTest.kt

```diff
@@ -146,23 +146,10 @@ class BeløpBeregnerTest {
         val tom = LocalDate.of(2024, 2, 15)
 
 
-        val result = BeløpBeregner.beløpForPeriode(BigDecimal("1002.50"), fom, tom)
-
-
-        val forventetBeløp = BigDecimal("2105") // Rundes ned fra 2105.25
-        result.shouldBe(forventetBeløp)
-    }
-
-    @Test
-    fun regnForPeriode_midtenAvDesember2023TilMidtenAvFebruar2024_rundesOpp_regnesKorrekt() {
-        val fom = LocalDate.of(2023, 12, 14)
-        val tom = LocalDate.of(2024, 2, 15)
-
-
         val result = BeløpBeregner.beløpForPeriode(BigDecimal("1002.25"), fom, tom)
 
 
-        val forventetBeløp = BigDecimal("2105") // Rundes opp fra 2104.73
+        val forventetBeløp = BigDecimal("2104.72") // Rundes ned fra 2104.7250
```

Comment:
Er ikke dette ulogisk test? Når treffer vi 2104.7250?

Eller er dette for å støtte mulige fremtidig/andre konsumenter?